.github/ISSUE_TEMPLATE/a-issue-report.yml

```
@@ -7,7 +7,7 @@ body:
       label: IMPORTANT
       description: Verify the following requirements have been met before opening an issue.
       options:
-        - label: My question is related to a new issue in the latest next release. For all other issues, open a ticket with [Esri Technical Support](https://support.esri.com/) post your question in the [community forum](https://community.esri.com/t5/arcgis-api-for-javascript/ct-p/arcgis-api-for-javascript), or share your idea with the global community on [ArcGIS Ideas](https://community.esri.com/t5/forums/searchpage/tab/message?advanced=false&allow_punctuation=false&q=javascript).
+        - label: My question is related to a new issue in the latest next release. For all other issues, open a ticket with [Esri Technical Support]( https://support.esri.com/en/contact-tech-support ) post your question in the [community forum](https://community.esri.com/t5/arcgis-api-for-javascript/ct-p/arcgis-api-for-javascript), or share your idea with the global community on [ArcGIS Ideas](https://community.esri.com/t5/forums/searchpage/tab/message?advanced=false&allow_punctuation=false&q=javascript).
```

remove extra spacing in the link & add comma
```suggestion
        - label: My question is related to a new issue in the latest next release. For all other issues, open a ticket with [Esri Technical Support](https://support.esri.com/en/contact-tech-support), post your question in the [community forum](https://community.esri.com/t5/arcgis-api-for-javascript/ct-p/arcgis-api-for-javascript), or share your idea with the global community on [ArcGIS Ideas](https://community.esri.com/t5/forums/searchpage/tab/message?advanced=false&allow_punctuation=false&q=javascript).
```